translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+        <source>Disconnection</source>
+        <target>Déconnection</target>
+      </trans-unit>
+      <trans-unit id="90245733925298447" datatype="html">
+        <source>Red pieces are called &quot;sources&quot;.
+        When a stack is not directly nor indirectly connected to a source, it is removed from the board.
+        You can move on a source: this forms a stack that contains the source, and which will never be disconnected.
+        Besides this, a stack containing a source acts like any other stack: you can move it, and your opponent can move on that stack to take possession of it.&lt;br/&gt;&lt;br/&gt;
+        You&apos;re playing Dark, move your piece on the source.</source>
+        <target>Les pièces rouges sont appelées “sources”.
+        Quand une pile n’est plus directement ou indirectement connectée à une source, elle est enlevée du plateau.&lt;br/&gt;&lt;br/&gt;
+        Vous jouez Foncé, déplacez votre pièce sur la source.</target>
+      </trans-unit>
+      <trans-unit id="1204343429877937229" datatype="html">
+        <source>Nice, you have disconnected 4 pieces of your opponent!
+        He has lost 4 points.
+        Disconnected stacks will not be visible at the next turn.</source>
+        <target>Bravo, vous avez déconnecté 4 pièces de l’adversaire !
+        Il a donc perdu 4 points.
+        Les piles déconnectées cesseront d&apos;être visibles au tour suivant.</target>
+      </trans-unit>
+      <trans-unit id="9072861571041891023" datatype="html">
+        <source>Bad choice! By moving on the source you would have disconnected the opponent&apos;s pieces and they would have lost 4 points.
+        Here, the opponent wins 2 to 0</source>
+        <target>Mauvais choix! En le déplaçant sur la source vous déconnectiez l&apos;adversaire et lui faisiez perdre ces 4 points.
+        Ici, il gagne 2 à 0.</target>
+      </trans-unit>
+      <trans-unit id="3483594998155999756" datatype="html">
+        <source>When no more move is possible, the game ends and the player with the most points wins.&lt;br/&gt;&lt;br/&gt;
+        Make your last move.</source>
+        <target>Quand plus aucun mouvement n’est possible, la partie est finie et le joueur avec le plus de points gagne.&lt;br/&gt;&lt;br/&gt;
+        Faites votre dernier mouvement !</target>
+      </trans-unit>
+      <trans-unit id="2963709509031109432" datatype="html">
+        <source>Congratulations, you won 6 - 0!</source>
+        <target>Bravo, vous avez même gagné 6 - 0 !</target>
+      </trans-unit>
+      <trans-unit id="8876232297721386956" datatype="html">
+        <source>Bad idea, by moving on the source you would have won a point.</source>
+        <target>Mauvaise idée, en la déplaçant sur la source, vous auriez gagné votre pièce et gagné un point.</target>
+      </trans-unit>
+      <trans-unit id="6986218395331151516" datatype="html">
+        <source>You must pick one of your pieces.</source>
```

Comment:
Duplicata avec "You must select one of your pieces."
Monomaniac-Refactoring :D

Reply:
je laisse en suspend pour investigation future-proche

Reply:
fait en local !